server/api.go

```diff
@@ -853,3 +855,54 @@ func (p *Plugin) updateSettings(w http.ResponseWriter, r *http.Request, userID s
 	resp, _ := json.Marshal(info.Settings)
 	w.Write(resp)
 }
+
+func (p *Plugin) getIssueByNumber(w http.ResponseWriter, r *http.Request, userID string) {
+
+	ctx := context.Background()
+
+	var githubClient *github.Client
+	owner := r.FormValue("owner")
+	repo := r.FormValue("repo")
+	number := r.FormValue("number")
```

Comment:
Other API methods are using a request body instead.  (e.g. https://github.com/mattermost/mattermost-plugin-github/blob/119ff1807a40eca316e5c1644589f056baa71936/server/api.go#L284-L291)

I would like to converge on that pattern for consistency despite there not being a functional difference.

Reply:
Well... I would disagree on this... `/createissuecomment` is a POST endpoint which calls the Github POST endpoint, so it makes sense that it receives a body. But in the case of `/issue` and `/pr` are GET endpoints so it makes sense to receive the values as query params.
What do you think?

Reply:
Good point, I completely missed that `/issue` and `/pr` are `POST` endpoints.